### ISSUE TITLE
What is the strategy for the hyperparameter settings in other keypoint detection tasks?

### ISSUE THREAD

**Issue by ziqi123** [princeton-vl/pytorch_stacked_hourglass#37]
Hi, I am new to the computer vision and recently working on the parking space detection job. Given a picture of a parking space, my interest is to identify the four corner points in it, namely the keypoints.

I have referenced your work of Hourglass network used for human pose estimation and taken advantage of your PyTorch code directly in my project. However, the accuracy is rather low.

I have noticed that Hourglass consists of many blocks and is set with some hyperparameters with regard to the convolution layers. So I wonder if you have some special strategies for the hyperparameter settings in your human pose estimation task, which might be useful for my own work.

Thank you for your patience!

**Comment by crockwell** [comment]
I can't recommend any special hyperparameter tuning for this task. My main advice would be to make sure you can get good training accuracy first and check for bugs in things like data augmentation and pre/post-processing, and that you are properly reloading the model from training.
